AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/LocalStorage/LocalStorageError.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Errors interfacing with local storage
+public enum LocalStorageError: Error {
```

Comment:
Is `public` needed or can it be `internal`?

Reply:
Good catch, I've updated to be internal.